### PR TITLE
cleanup: cleanup e2e of mcs

### PR DESCRIPTION
**Pull request by jwcesign**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
cleanup: cleanup e2e of mcs

**Which issue(s) this PR fixes**:
Fixes #none

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: test/e2e/mcs_test.go [hunk 1]
```diff
@@ -841,36 +794,24 @@ func checkEndpointSliceSynced(provisionEPSList, consumptionEPSList *discoveryv1.
...
 
-	synced := false
+	var synced bool
 	for _, item := range provisionEPSList.Items {
 		if item.GetLabels()[discoveryv1.LabelManagedBy] == util.EndpointSliceDispatchControllerLabelValue {
 			continue
 		}
+		synced = false
 		for _, consumptionItem := range consumptionEPSList.Items {
+			klog.Infof("jw1:%v,%v/%v,%v", consumptionItem.Name, len(consumptionItem.Endpoints), item.Name, len(item.Endpoints))
```
Comment (XiShanYongYe-Chang) [inline test/e2e/mcs_test.go:804]: debug info.

File: test/e2e/mcs_test.go [hunk 2]
```diff
@@ -841,36 +794,23 @@ func checkEndpointSliceSynced(provisionEPSList, consumptionEPSList *discoveryv1.
 		return true
 	}
 
-	synced := false
+	var synced bool
 	for _, item := range provisionEPSList.Items {
 		if item.GetLabels()[discoveryv1.LabelManagedBy] == util.EndpointSliceDispatchControllerLabelValue {
 			continue
 		}
+		synced = false
```
Comment (XiShanYongYe-Chang) [inline test/e2e/mcs_test.go:802]: How about this: 
```go
synced := false
```

File: test/e2e/mcs_test.go [hunk 3]
```diff
@@ -841,36 +794,23 @@ func checkEndpointSliceSynced(provisionEPSList, consumptionEPSList *discoveryv1.
 		return true
 	}
 
-	synced := false
+	var synced bool
```
Comment (XiShanYongYe-Chang) [inline test/e2e/mcs_test.go:797]: This line is no more needed.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Thanks~
Just waiting for the CI pass.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
/lgtm
/approve

**Comment by RainbowMango** [comment]
> cleanup: cleanup e2e of mcs

Why do we need it?  flaking test?

**Pull request merged**